code/src/components/End.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+export const End = ({ name, yesno, shape }) => {
+  // console.log(name,yesno,shape, "--------")
+  if (yesno === 'No') {
+    yesno = 'Just like you';
+  } else if (yesno === 'Yes') {
+    yesno = 'Unlike you';
+  }
```

Comment:
Smart to add an if/else if to change the text depending on the answers from the user!